cpu/numactl.py

```diff
@@ -193,6 +194,8 @@ def test_localalloc(self):
         '''
         Test memory allocation on the current node
         '''
+        if not self.iface and not self.disk:
```

Comment:
Should this be OR instead and as at any point I will be running either NIC NUma woth only interface input or FC NUMA with disk as input

Can this be taken care in setup instead in test ?

Reply:
The condition for cancel is if both iface and disk values are not specified by user. It cannot be OR as all the tests (except first) would then be cancelled.
If the user provides any one or both the values then the corresponding tests will be executed.

I cannot add this check in setup as that would cancel first test also (which was the original problem this fix addresses)